mne_lsl/lsl/load_liblsl.py

```diff
@@ -59,12 +60,39 @@ def load_liblsl() -> CDLL:
             "provide the error traceback to the developers."
         )
     libpath = _load_liblsl_environment_variables()
+    libpath = _load_liblsl_wheel_path() if libpath is None else libpath
     libpath = _load_liblsl_system() if libpath is None else libpath
     libpath = _load_liblsl_mne_lsl() if libpath is None else libpath
     libpath = _fetch_liblsl() if libpath is None else libpath
```

Comment:
In the end, we would remove all the custom code for system liblsl and downloaded liblsl, correct? Thus leaving:

```suggestion
    libpath = _load_liblsl_environment_variables()
    libpath = _load_liblsl_wheel_path() if libpath is None else libpath
```

Reply:
Yes that's the idea!

I won't push this commit as I think a bunch of tests will need to be modified, too. I can give it a shot but also happy if you do it -- you wrote the tests so you could probably more quickly cut them down

Reply:
Done (but I broke something, looking into it!)